### PR TITLE
Adding resources to test operatorhub bundle locally

### PR DESCRIPTION
**Pull request by RameshaKaru**
This PR adds the resources to test operator bundle locally

Related PR contribution: https://github.com/operator-framework/community-operators/pull/917

### CODE REVIEW

File: operatorhub-resources/apim-operator/1.0.1/apim-operator.v1.0.1.clusterserviceversion.yaml [hunk 1]
```diff
@@ -0,0 +1,276 @@
+apiVersion: operators.coreos.com/v1alpha1
```
Comment (dinusha92) [inline operatorhub-resources/apim-operator/1.0.1/apim-operator.v1.0.1.clusterserviceversion.yaml]: add license headers.

File: operatorhub-resources/apim-operator/1.0.1/ratelimitings.wso2.com.crd.yaml [hunk 2]
```diff
@@ -0,0 +1,108 @@
+apiVersion: apiextensions.k8s.io/v1beta1
```
Comment (dinusha92) [inline operatorhub-resources/apim-operator/1.0.1/ratelimitings.wso2.com.crd.yaml]: Add license headers.

File: operatorhub-resources/install.yaml [hunk 3]
```diff
@@ -0,0 +1,29 @@
...
+apiVersion: operators.coreos.com/v1alpha1
+kind: Subscription
+metadata:
+  name: my-operator-subsription
+  namespace: marketplace
+spec:
+  channel: stable
+  name: apim-operator
+  source: rameshakaru-operators
+  sourceNamespace: marketplace
```
Comment (dinusha92) [inline operatorhub-resources/install.yaml]: add a new line at the end.

File: operatorhub-resources/apim-operator/1.0.1/ratelimitings.wso2.com.crd.yaml (hunk 2 above)
Comment (RameshaKaru) [reply operatorhub-resources/apim-operator/1.0.1/ratelimitings.wso2.com.crd.yaml]: Added

File: operatorhub-resources/apim-operator/1.0.1/apim-operator.v1.0.1.clusterserviceversion.yaml (hunk 1 above)
Comment (RameshaKaru) [reply operatorhub-resources/apim-operator/1.0.1/apim-operator.v1.0.1.clusterserviceversion.yaml]: Added

File: operatorhub-resources/apim-operator/1.0.1/ratelimitings.wso2.com.crd.yaml (hunk 2 above)
Comment (RameshaKaru) [reply operatorhub-resources/apim-operator/1.0.1/ratelimitings.wso2.com.crd.yaml]: Added

File: operatorhub-resources/install.yaml (hunk 3 above)
Comment (RameshaKaru) [reply operatorhub-resources/install.yaml]: Added

**Pull request merged**